rails-install-ubuntu.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
-curl -sL https://deb.nodesource.com/setup_dev | sudo bash -
```

Comment:
The reason I wanted to remove the dependency on curl was because the script itself attempts to install curl. So I probably should have went the other route and removed it from the list of things apt is supposed to install. Another route would be to change the README instructions for ubuntu, but I guess I was lazy.